examples/pages/basic/index.js

```diff
@@ -108,23 +108,30 @@ function BasicAutocomplete({items, onChange}) {
   return (
     <Autocomplete onChange={onChange}>
       <Input placeholder="Favorite color ?" />
-      <Autocomplete.Menu style={{border: '1px solid rgba(34,36,38,.15)'}}>
-        {({inputValue, selectedItem, highlightedIndex}) =>
-          // prettier is doing weeeeird things to this
-          // prettier-ignore
-          items.filter(i => !inputValue || i.toLowerCase().includes(inputValue.toLowerCase()))
-            .map((item, index) => (
-              <Item
-                value={item}
-                index={index}
-                key={item}
-                highlightedIndex={highlightedIndex}
-                selectedItem={selectedItem}
-              >
-                {item}
-              </Item>
-            ))}
-      </Autocomplete.Menu>
+      <Autocomplete.Menu
+        style={{border: '1px solid rgba(34,36,38,.15)'}}
+        render={({isOpen, inputValue, selectedItem, highlightedIndex}) =>
+          isOpen &&
```

Comment:
I mentioned this a little above, but basically it allows for things [like this](https://github.com/moroshko/react-autosuggest#alwaysRenderSuggestionsProp) as well as more control when using animations. It will also help when creating a dropdown that contains an input inside the menu.